distro/docker/src/main/docker/Dockerfile.native

```diff
@@ -1,13 +1,14 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal
-
-ENV APP_URL=app/target/apicurio-registry-app-*-runner
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3
 
 WORKDIR /work/
 
-ADD ${APP_URL} /work/application
-
-RUN chmod 775 /work
+RUN chown 1001 /work \
+    && chmod "g+rwX" /work \
+    && chown 1001:root /work
+    
+COPY --chown=1001:root target/apicurio-registry-*-runner /work/application
 
 EXPOSE 8080
+USER 1001
 
 CMD ["./application", "-Dquarkus.http.host=0.0.0.0"]
```

Comment:
No more `JAVA_OPT` - but I assume ENV vars still work...

Reply:
env vars for everything now